pkg/kudoctl/kudoinit/prereq/webhook.go

```diff
@@ -70,6 +82,46 @@ func (k kudoWebHook) AsRuntimeObjs() []runtime.Object {
 	return objs
 }
 
+func validateCertManagerInstallation(client *kube.Client) kudoinit.Result {
+	if result := validateCrdVersion(client.ExtClient, "certificates.cert-manager.io", certManagerAPIVersion); !result.IsEmpty() {
+		return result
+	}
+	if result := validateCrdVersion(client.ExtClient, "issuers.cert-manager.io", certManagerAPIVersion); !result.IsEmpty() {
+		return result
+	}
+
+	deployment, err := client.KubeClient.AppsV1().Deployments("cert-manager").Get("cert-manager", metav1.GetOptions{})
+	if err != nil {
+		return kudoinit.NewWarning(fmt.Sprintf("failed to get cert-manager deployment in namespace cert-manager. Make sure cert-manager is running (%s)", err))
+	}
+	if len(deployment.Spec.Template.Spec.Containers) < 1 {
+		return kudoinit.NewWarning("failed to validate cert-manager controller deployment. Spec had no containers")
+	}
+	if !strings.HasSuffix(deployment.Spec.Template.Spec.Containers[0].Image, "cert-manager-controller:v0.12.0") {
+		return kudoinit.NewWarning(fmt.Sprintf("cert-manager deployment had unexpected version. expected v0.12.0 in controller image name but found %s", deployment.Spec.Template.Spec.Containers[0].Image))
+	}
+	if deployment.Status.ReadyReplicas < 1 {
```

Comment:
what do you think about using health module for this? I think it's something like `health.IsHealthy` and you can pass in deployment

Reply:
Good idea, done.